.github/workflows/test-rust.yaml

```diff
@@ -37,6 +37,7 @@ jobs:
         with:
           crate: cargo-insta
       - run: ./.github/set_version.sh
+        shell: bash
```

Comment:
Are there other workflows that we run on Windows where we should set this as well?

Reply:
I'll leave this to you to do in #3119 and reduce this PR to the quote fix.